mlops_team_project/models/xgboost_model.py

```diff
@@ -140,12 +168,28 @@ def model(
 
     logging.info(classification_report(y_test, preds, target_names=target_names))
 
-    with open("models/xgboost_model.pkl", "wb") as file:
-        pickle.dump(model, file)
+    save_model_to_google(model)
 
     return ModelResponse(train_accuracy, test_accuracy)
 
 
+def read_from_google(file_name: str):
```

Comment:
Note to self @henry-debuchananne 

This is where we will add the DVC communication